src/components/SectorPolygons.jsx

```diff
@@ -6,7 +6,7 @@ import { Layer, Source } from 'react-map-gl';
 import { configurationStore, cwpStore } from '../state';
 
 const sectorOutlinePaint = {
-  'line-color': '#fff',
```

Comment:
I think the property could be named `line-color` or `color` (or `colour`) instead of `description`.